EVEStandard/Enumerations/TransactionType.cs

```diff
@@ -1,128 +1,130 @@
 ﻿namespace EVEStandard.Enumerations
 {
     /// <summary>
-    /// Transaction type, different type of transaction will populate different fields in `extra_info` Note: If you have an existing XML API application that is using ref_types, you will need to know which string ESI ref_type maps to which integer. You can use the following gist to see string->int mappings: https://gist.github.com/ccp-zoetrope/c03db66d90c2148724c06171bc52e0ec
+    /// Transaction type, different type of transaction will populate different fields in `extra_info` Note: If you have an existing XML API application that is using ref_types,you will need to know which string ESI ref_type maps to which integer. You can use the following gist to see string->int mappings: https://gist.github.com/ccp-zoetrope/c03db66d90c2148724c06171bc52e0ec
     /// </summary>
     /// <value>Transaction type, different type of transaction will populate different fields in `extra_info` Note: If you have an existing XML API application that is using ref_types, you will need to know which string ESI ref_type maps to which integer. You can use the following gist to see string->int mappings: https://gist.github.com/ccp-zoetrope/c03db66d90c2148724c06171bc52e0ec</value>
     public enum TransactionType
```

Comment:
I guess we don't need the int references anymore since people should be using the strings returned to convert? Do we need that json string property I think we have on other enums that are really strings instead of ints?

Reply:
What i mean by the second question is this. [JsonConverter(typeof(StringEnumConverter))]

Reply:
I think we do. If it's not there it gets converted to INT and the ESI requires a string.

Reply:
Once you add the JsonConverter part I'll be happy to approve the PR.